app/src/main/java/com/google/android/samples/socialite/data/AppDatabase.kt

```diff
@@ -25,21 +25,25 @@ import com.google.android.samples.socialite.model.Chat
 import com.google.android.samples.socialite.model.ChatAttendee
 import com.google.android.samples.socialite.model.Contact
 import com.google.android.samples.socialite.model.Message
+import com.google.android.samples.socialite.widget.model.WidgetModel
+import com.google.android.samples.socialite.widget.model.WidgetModelDao
 
 @Database(
     entities = [
         Contact::class,
         Chat::class,
         ChatAttendee::class,
         Message::class,
+        WidgetModel::class,
     ],
     views = [ChatWithLastMessage::class],
-    version = 1,
+    version = 2,
```

Comment:
You can keep it `1`. Other members can just uninstall/install the app.

Reply:
Fixed